lambdas/time-epoch-handler/src/time-handler.ts

```diff
@@ -1,29 +1,27 @@
 import { LambdaInterface } from "@aws-lambda-powertools/commons";
 import { TimeEvent } from "./time-event";
-import { toEpochSecondsFromNow } from "./utils/date-time";
+import { milliseconds, toEpochSecondsFromNow } from "./utils/date-time";
 import { LogHelper } from "../../logging/log-helper";
 import { Context } from "aws-lambda";
 
-const logHelper = new LogHelper();
-
 export class TimeHandler implements LambdaInterface {
+  constructor(private readonly logger: LogHelper = new LogHelper()) {}
   public async handler(event: TimeEvent, context: Context) {
-    logHelper.logEntry(context.functionName, event.govJourneyId);
+    this.logger.logEntry(context.functionName, event.govJourneyId);
 
-    if (event.ttlValue < 0) {
+    if (Number(event.ttlValue) < 0) {
```

Comment:
So the concern is that we could potentailly pass in a negative TTL? Why is that? I wouldn't expect `toEpochSecondsFromNow` to throw if we pass in a negative duration

Reply:
You are right, but why would we want to pass a negative duration? This just codifies that we will never intentionally pass negative duration. I think at some point i was doing `Math.abs` what do you think?

Reply:
Well I guess it's up to the consumer to decide if they want to pass a negative duration, right? Maybe they're trying to get a past time for something for example a date has to be x days in the past

If we're thinking purely from a time handler perspective - it doesn't know what context it's going to be used in and it shouldn't really care?

Reply:
I'm not sure I'm following - there is more code when we don't allow for negatives, right? What extra work would need to be done to accommodate this apart from removing the if statement? 

Just keen to make sure we're not adding extra complexity to the code by trying to fix problems that don't exist yet